commands/RuneScape - Fun/raid.js

```diff
@@ -0,0 +1,189 @@
+const { Command } = require('klasa');
+const RaidsEmojis = require('../../resources/monsters/raids').drops;
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			cooldown: 1,
+			description: 'Roll on the raids drop table based on the number of points specified',
+			usage: '<points:int{1}>',
+			usageDelim: ' '
```

Comment:
you can completely delete the usageDelim line, as there is only 1 input parameter, theres nothing to Deliminate